### DISCUSSION TITLE
Tutorial tamplates and examples.

### DISCUSSION THREAD

**Discussion by JosipKustan** [AlgebraStudentCollab/START-HERE#15] [Ideas]
I had some time to make a quick  "How to git/How to use our Repos" type a deal.

Questions:
What would be a natural place to put them? 
What do you have to say on the first tutorial? [here](https://github.com/AlgebraStudentCollab/START-HERE/blob/main/Git%20and%20How%20To%20Start.md) (its not fully done. But I need a feeling check. I plan to stylelize the tamplate so we can reuse it, and add some usefull links)
I was thinking about recording quick HowTos. Its quicker for me then writing. wdys?

@lcabraja @tsucevic @mblaze2 @mkovac6  @psporcic

**Comment by marijo-blazevic** [comment]
1. Maybe create another repo how-to, like code-of-conduct, and just link/reference it in every repo..
2. Awesome! simple and effective
3. Sure, videos are cool. Although, markdowns are easy to edit in the future is something changes with how we use our repos ..

**Reply by JosipKustan** [reply]
So we make a repo "tutorials" and we link from any repo to some relevant tutorial file.
Is it ok if we put them in main branch -> folder tutorials -> collection(Git, Python, Java) -> tutorialfile.md?
